### ISSUE TITLE
Adjust poster and thumbnail size on home screen

### ISSUE THREAD

**Issue by MRobi1** [jellyfin/jellyfin-androidtv#3072]
### Describe the feature you'd like

Would be nice to see an option to adjust the poster and thumbnail sizes on the home screen. This can currently be done in the library views with poster size of small, medium, large, XL. I find on the home screen the Library images and thumbnails to be larger than the posters, so some customizability here as well would be a nice add and a feature I don't believe any competitors offer.

**Comment by nielsvanvelzen** [comment]
I agree the items on the home screen are currently too big. I've looked into making them smaller not that long ago but unfortunately there are too many hardcoded numbers in the home screen making it hard to do right. It might need a rewrite of the home screen first so the code isn't too intertwined with other parts of the app.

So yeah good idea but might take some time before we can implement it.

**Comment by CiaranDougherty** [comment]
Trickiness (and low priority) is understood, just adding my interest in this enhancement

**Comment by bernieol** [comment]
css code i found ,works a treat on jellyfin media server .but no way to input that same code into client software
unless i wish to spend a week inputting it with tv remote ,one letter at a time (painful).
 posters are nice ,just too big .
